android/build.gradle

```
@@ -20,7 +22,7 @@ def safeExtGet(prop, fallback) {
 }
 
 android {
-    compileSdkVersion safeExtGet('compileSdkVersion', 28)
+    compileSdkVersion safeExtGet('compileSdkVersion', 29)
```

```suggestion
    compileSdkVersion safeExtGet('compileSdkVersion', 30)
```

we should be able to compile against 30 now?

---

yes, if this configuration works with defaults in your project